modules/core/src/controllers/map-controller.js

```diff
@@ -306,6 +306,21 @@ class MapState extends ViewState {
     });
   }
 
+  shortestPathFrom(viewState) {
+    // const endViewStateProps = new this.ControllerState(endProps).shortestPathFrom(startViewstate);
+    const fromProps = viewState.getViewportProps();
```

Comment:
Nit: This may be temporary, but to avoid confusion, I think we should only use `viewState` for the plain javascript object, this variable should be `controllerState` or similar.

Reply:
In the same vein, the `getViewportProps()` should probably be named `getViewState()`

Reply:
I'll do this in a follow up PR, as `getViewportProps` is not introduced by this change.